tests/lib/rules/arrow-body-style.js

```diff
@@ -72,6 +84,7 @@ ruleTester.run("arrow-body-style", rule, {
         },
         {
             code: "var foo = () => { return bar(); };",
+            output: "var foo = () =>   bar() ;",
```

Comment:
Yeah, I think we have to special case this. I can't see anyone being happy with more than one space after `=>`.